### PR TITLE
SW-2687 Temporary time zones hook to unblock selector component and other page views

### PR DESCRIPTION
**Pull request by karthikbtf**
- this is temporary
- localization provider will replace this with it's time zones

### CODE REVIEW

**Review by charlie-tf — APPROVED** [review]
Looks good. Thanks for doing this

**Review by constanzauanini — APPROVED** [review]
Tests are failing 😕

**Comment by karthikbtf** [comment]
> Tests are failing 😕

I saw that, what does it have to do with my change though 🤯

File: src/providers/hooks.ts [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 import { OrganizationContext, UserContext } from './contexts';
 
 export const useUser = () => useContext(UserContext);
 
+/**
+ * Default time zones until we have a provider.
+ */
+export const useTimeZones = () =>
+  useMemo(() => {
+    const defaultTimeZones = (Intl as any).supportedValuesOf('timeZone').map((tz: any) => ({ id: tz, longName: tz }));
```
Comment (karthikbtf) [inline src/providers/hooks.ts:11]: moving this into the function seems to fix the tests error.. not sure how related.
will get this in for now and see how it works.

**Pull request merged**